test/integration/graql/reasoner/query/NegationIT.java

```diff
@@ -660,10 +667,10 @@ public void allRecipesContainingAvailableIngredients(){
 
             assertCollectionsNonTriviallyEqual(recipesWithAllIngredientsAvailableExplicit, recipesWithAllIngredientsAvailable);
             assertCollectionsNonTriviallyEqual(recipesWithAllIngredientsAvailableExplicit, recipesWithAllIngredientsAvailableSimple);
-            assertCollectionsNonTriviallyEqual(recipesWithAllIngredientsAvailable, ListsUtil.listDifference(allRecipes, recipesWithUnavailableIngredients));
         }
     }
 
```

Comment:
sooooo loads of tests have appeared in `verification`, but not much deleted here... missing a commit?

Reply:
At the time of review there wasn't a test in verification that tested two nested negations with conjunctions. I have since added one and deleted this test here.